resources/js/components/send/FormField.vue

```diff
@@ -1,32 +1,34 @@
 <template>
   <div>
-    <div class="card mb-6 overflow-hidden">
-      <div class="flex w-2/3 justify-between px-8 pt-4">
-        <div
-          class="help-text mr-4"
-        >Adjust an existing mail template or craft an email from scratch! You can use normal blade syntax and include attributes from this resources model&hellip;</div>
-      </div>
-
-      <div class="flex border-b border-40 remove-bottom-border px-8 py-4">
-        <div class="w-full pb-2">
+    <div class="py-3 px-8">
+      <p class="leading-normal help-text">
+        Adjust an existing mail template or craft an email from scratch! You can use normal blade syntax and include attributes from this resources model&hellip;
+      </p>
+    </div>
+    <div v-if="hasTemplates">
+      <DefaultField :field="field" fieldName="Mail Template">
```

Comment:
It's a bit odd reusing "field" since that represents the "send-mail" field, but it seems to work fine since we aren't using most of the inner workings and are mainly just using it for HTML structure.